api/src/main/resources/messages.properties

```diff
@@ -631,6 +631,8 @@ reporting.parameter.startDate=Start Date
 reporting.parameter.endDate=End Date
 reporting.parameter.location=Location
 reporting.parameter.locationList=Locations
+reporting.parameter.attributeType=Attribute Type
+reporting.parameter.values=Value
```

Comment:
They are two different fields, the attribute type determines the particular attribute you want to use for the search e.g Birthplace, Citizenship e.t.c while the value specifies the search query, e.g for Citizenship the value may be Nigeria so that will retrieve all patients/people whose citizenship is Nigeria
![screen shot 2017-03-31 at 11 22 12 am](https://cloud.githubusercontent.com/assets/17332992/24546696/61470658-1604-11e7-8bd8-1de6ba532991.png)